cmd/hzn/main.go

```diff
@@ -143,6 +144,12 @@ func StartHealthz(L hclog.Logger) {
 		w.WriteHeader(200)
 	})
 
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
```

Comment:
In our other products we usually hide these behind AuthN/AuthZ. Should we perhaps do the same here?

Reply:
These are only running on the metrics endpoint, which is never exposed.

Reply:
Metrics run a separate port with separate http listener.

Reply:
Got it!